Gaijinizer patch/Troops.txt

```diff
@@ -9927,19 +9927,19 @@ Hyakki final processing#MTLed
 "◆百鬼◆\\n『もう、後戻りはできないヨ』"
 > CONTEXT: Troops/254/pages/1/17/InlineScript/1:6
 > CONTEXT: Troops/256/pages/6/115/InlineScript/1:6
-"◆ Hyakki ◆ n n "I can not go back, anymore""#MTLed
+"◆ Hyakki ◆ n n "There's no going back now""#Ron
 > END STRING
```

Comment:
This is an "InlineScript". This might break things if not formatted correctly. It should be like this:
>_**"◆ Hyakki ◆ \\n [There's no going back now.]"**_